app/views/admin/configs/show.html.erb

```diff
@@ -175,30 +178,27 @@
                     <%= inline_svg_tag("email.svg", class: "crayons-icon", aria: true, title: "Twitter logo") %>
                   </figure>
                   <div class="config-authentication__item--container">
-                    <div class="config-authentication__item--label">
-                      <p class="fw-medium">
-                        Email address and password
-                      </p>
+                    <div style="display: flex; align-content: center; justify-content: space-between; align-items: center;">
+                      <div style="display: flex; align-content: center; justify-content: space-between; align-items: center;">
+                        <p class="config-authentication__item--label fw-medium">
+                          Email address
+                        </p>
+                      </div>
+                      <button
+                        class="crayons-btn crayons-btn--secondary"
+                        data-button-text="<%= SiteConfig.allow_both_email_signup_and_login ? "edit" : "enable" %>"
+                        data-target="config.emailAuthSettingsBtn"
```

Comment:
In order to fulfill the design specs requirement of a button for enabling/disabling Email Auth, I employ the hidden `allow_both_email_signup_and_login` field to pass as a param, which then updates the fields actually relevant to email auth in the controller.

I did consider sticking with the already-existing checkboxes for `allow_email_password_registration` and `allow_email_password_login`, and simply hiding those, but I decided my implementation was cleaner.

Reply:
I agree with this approach: it's clear in the controller that it's translating this value into database values.